### ISSUE TITLE
Error with node 0.11.15

### ISSUE THREAD

**Issue by vanthome** [bnoordhuis/node-heapdump#49]
I cannot install with node 0.11.15:

````
> heapdump@0.3.4 install /server/node_modules/heapdump
> node-gyp rebuild

child_process: customFds option is deprecated, use stdio instead.
make: Entering directory '/server/node_modules/heapdump/build'
  CXX(target) Release/obj.target/addon/src/heapdump.o
In file included from ../src/heapdump.cc:17:0:
../src/compat-inl.h: In static member function ‘static void compat::Isolate::SetAddHistogramSampleFunction(v8::Isolate*, v8::AddHistogramSampleCallback)’:
../src/compat-inl.h:100:3: error: ‘SetAddHistogramSampleFunction’ is not a member of ‘v8::V8’
   v8::V8::SetAddHistogramSampleFunction(callback);
   ^
../src/compat-inl.h: In static member function ‘static void compat::Isolate::SetCreateHistogramFunction(v8::Isolate*, v8::CreateHistogramCallback)’:
../src/compat-inl.h:110:3: error: ‘SetCreateHistogramFunction’ is not a member of ‘v8::V8’
   v8::V8::SetCreateHistogramFunction(callback);
   ^
addon.target.mk:83: recipe for target 'Release/obj.target/addon/src/heapdump.o' failed
make: *** [Release/obj.target/addon/src/heapdump.o] Error 1
make: Leaving directory '/server/node_modules/heapdump/build'
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/lib64/node_modules/npm/node_modules/node-gyp/lib/build.js:267:23)
gyp ERR! stack     at ChildProcess.emit (events.js:110:17)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:1067:12)
gyp ERR! System Linux 3.18.2n-fuse
gyp ERR! command "node" "/usr/lib64/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /server/node_modules/heapdump
gyp ERR! node -v v0.11.15
gyp ERR! node-gyp -v v1.0.2
gyp ERR! not ok 
````

**Comment by bnoordhuis** [comment]
@vanthome Thanks for the bug report.  Fixed in b9a059d and published as v0.3.5.

**Comment by vanthome** [comment]
cool, many thanks
